include/aspect/velocity_boundary_conditions/gplates.h

```diff
@@ -333,40 +222,39 @@ namespace aspect
 
       private:
         /**
-         * A variable that stores the current time of the simulation, but
-         * relative to the velocity_file_start_time.
+         * A variable that stores the currently used data file of a series. It
+         * gets updated if necessary by update().
          */
-        double time_relative_to_vel_file_start_time;
+        int  current_file_number;
```

Comment:
Could we make this an unsigned int (because conceptually a file number is a non-negative number)? If you need an invalid value, you can always use `numbers::invalid_unsigned_int`.